### DISCUSSION TITLE
queryRaw, use AS alias on dynamic column.

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#9698] [Q&A] [answered]
this query was working fine before prsima version 3.0.1

```ts
const identifierType = 'LocalCode';
const value = 'AAPL';
const results  = prisma.$queryRaw(`SELECT  ${identifierType} as 'requestedIdentifier' ,  CUSIP, ISIN, SEDOL, LocalCode FROM tableName WHERE ${identifierType}=${value};`);

```
In latest prisma, prisma.$queryRaw only support Tagged Template, direct conversion to tagged template isn't working.
This is after tagged Template:
```ts
const identifierType = 'LocalCode';
const value = 'AAPL';
const results  = prisma.$queryRaw`SELECT  ${identifierType} as 'requestedIdentifier' ,  CUSIP, ISIN, SEDOL, LocalCode FROM tableName WHERE ${identifierType}=${value};`;

```

but this is not evaluating as it was in first scenario.
When i log query i get this:
SELECT ? as 'requestedIdentifier', CUSIP, ISIN, LocalCode FROM tableName where ?=?;

one way is to use prisma.$rawQueryUnsafe(``);

But How can make $rawQuery'' Tagged Template to work in my case.?
Thanks.

**Comment by ryands17** [comment, marked as answer]
@manishzversal 👋 

You cannot use `$queryRaw` for dynamic columns. In this case, the only option is to sanitise your input for the request and use `$queryRawUnsafe`.

**Reply by ghost** [reply]
ok Thanks @ryands17.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
